File: cw/cpp/remove_exclamation_marks.cpp
// https://www.codewars.com/kata/57a0885cbb9944e24c00008e/train/cpp


#include <iostream>
#include <regex>

std::string removeExclamationMarks(std::string str) {
    std::regex mark("[!]");
    return std::regex_replace(str, mark, "");
}

int main() {
    std::string str;

    std::cin >> str;
    std::cout << removeExclamationMarks(str) << std::endl;

    return 0;
}

File: codewars/elixir/remove_exclamation_marks.exs
# https://www.codewars.com/kata/57a0885cbb9944e24c00008e/train/elixir

defmodule Codewars do
  def remove_exclamation_marks(s) do
    String.replace(s, "!", "")
  end
end

IO.puts(Codewars.remove_exclamation_marks("Hello World!!!"))
